internal/controller/mysqluser_controller.go

```diff
@@ -303,3 +314,20 @@ func (r *MySQLUserReconciler) ifOwnerReferencesContains(ownerReferences []metav1
 	}
 	return false
 }
+
+func (r *MySQLUserReconciler) updateGrants(ctx context.Context, mysqlClient *sql.DB, mysqlUserName string, mysqlUserHost string, grants []mysqlv1alpha1.Grant) error {
+	log := log.FromContext(ctx)
+	_, err := mysqlClient.ExecContext(ctx, fmt.Sprintf("REVOKE ALL PRIVILEGES, GRANT OPTION FROM '%s'@'%s';", mysqlUserName, mysqlUserHost))
```

Comment:
I think if revoking all privileges in every reconciliation loop will make the the user unstable. So I'd like to consider other way.